website/sidebars.js

```diff
@@ -307,6 +307,7 @@ const sidebarSettings = {
         "reference/resource-configs/materialize-configs",
         "reference/resource-configs/firebolt-configs",
         "reference/resource-configs/teradata-configs",
+        "reference/resource-configs/clickhouse-configs",
         "reference/resource-configs/mindsdb-configs"
         "reference/resource-configs/mssql-configs",
```

Comment:
@dataders Line 311 here is missing a comma, I believe that is the source of the failed build